ground/gcs/src/app/main.cpp

```diff
@@ -301,11 +270,48 @@ int main(int argc, char **argv)
     parser.addPositionalArgument(
         "config", QCoreApplication::translate("main", "Use the specified configuration file."),
         QCoreApplication::translate("main", "config file"));
+
     if (!parser.parse(QCoreApplication::arguments())) {
         displayError(parser.errorText());
         displayHelpText(parser.helpText());
         return 1;
     }
+
+#ifdef USE_CRASHREPORTING
+    // Only use crash reporter if we're not doing unit tests
+    // This means we won't hang CI, and QtTest will give us a nice stack trace
+    if (parser.values(doTestsOption).isEmpty()) {
```

Comment:
This is the new part here. The rest is just moving the argument parsing above init of crash reporter.